encryptonize.go

```diff
@@ -86,76 +90,99 @@ func New(keyProvider key.Provider) (Encryptonize, error) {
 ////////////////////////////////////////////////////////
 
 // Encrypt creates a new sealed object containing the provided plaintext data as well as an access
-// object that controls access to that data. The calling user is automatically added to the access
-// object. To grant other users access, see AddGroupsToAccess and AddUserToGroups.
+// list that controls access to that data. The calling user is automatically added to the access
+// list. To grant other users access, see AddGroupsToAccess and AddUserToGroups.
 //
 // For all practical purposes, the size of the ciphertext in the SealedObject is len(plaintext) + 48
 // bytes.
-//
-// The sealed object and the sealed access object are not sensitive data.
-func (e *Encryptonize) Encrypt(user *data.SealedUser, object *data.Object) (data.SealedObject, data.SealedAccess, error) {
+func (e *Encryptonize) Encrypt(user *data.SealedUser, object *data.Object) (uuid.UUID, error) {
```

Comment:
We should say in the documentation here what the returned UUID represents, i.e., that the returned UUID is the key used by the IO provider to identify the SealedObject.